conf/cassandra.yaml

```diff
@@ -1033,36 +1038,36 @@ request_timeout_in_ms: 10000
 # The global limit is imposed on all messages exceeding the per-link limit, simultaneously with the per-endpoint limit,
 # on all links to or from any node in the cluster.
 #
-# internode_application_send_queue_capacity_in_bytes: 4194304                       #4MiB
-# internode_application_send_queue_reserve_endpoint_capacity_in_bytes: 134217728    #128MiB
-# internode_application_send_queue_reserve_global_capacity_in_bytes: 536870912      #512MiB
-# internode_application_receive_queue_capacity_in_bytes: 4194304                    #4MiB
-# internode_application_receive_queue_reserve_endpoint_capacity_in_bytes: 134217728 #128MiB
-# internode_application_receive_queue_reserve_global_capacity_in_bytes: 536870912   #512MiB
+# internode_application_send_queue_capacity: 4194304b                       #4MiB
+# internode_application_send_queue_reserve_endpoint_capacity: 134217728b    #128MiB
+# internode_application_send_queue_reserve_global_capacity: 536870912b      #512MiB
+# internode_application_receive_queue_capacity: 4194304b                    #4MiB
+# internode_application_receive_queue_reserve_endpoint_capacity: 134217728b #128MiB
+# internode_application_receive_queue_reserve_global_capacity: 536870912b   #512MiB
```

Comment:
nit/reminder: Move to the more readable versions on the right if we can work out the virtual table back-compat issues.